src/js/components/stock-movement/outbound/StockMovementOutboundList.jsx

```diff
@@ -60,8 +61,12 @@ const StockMovementOutboundList = (props) => {
     const queryProps = queryString.parse(props.history.location.search);
     // IF VALUE IS IN A SEARCH QUERY SET DEFAULT VALUES
     if (queryProps.requisitionStatusCode) {
-      defaultValues.requisitionStatusCode = props.requisitionStatuses
-        .filter(({ value }) => queryProps.requisitionStatusCode.includes(value));
```

Comment:
This is a refactor that I had to implement in many places of the new List Table pages.
The problem was the used `include(...)` method.
Unfortunately I did not consider this before but this method does not return a exact match  but as it implies it returns true **if item is included in a value**.

basicaly if we have a list `['PARTIALLY_RECEIVED', 'PENDING'].includes('RECEIVED')` it would return `true`